pkgs/applications/editors/vim/plugins/deprecated.json

```
@@ -19,6 +19,10 @@
         "date": "2022-10-20",
         "new": "neodev-nvim"
     },
+    "nvchad-extensions": {
+      "date": "2023-08-19",
+      "new": "nvchad-ui",
```

this breaks `builtins.fromJSON`
```suggestion
      "new": "nvchad-ui"
```